client/src/components/Common/models/exportRecordModel.ts

```diff
@@ -62,17 +62,24 @@ export class ExportParamsModel implements ExportParams {
         return Boolean(this._params?.include_hidden);
     }
 
-    public equals(otherExportParams?: ExportParamsModel) {
+    public equals(otherExportParams?: ExportParams) {
         if (!otherExportParams) {
             return false;
         }
-        return (
-            this.modelStoreFormat === otherExportParams.modelStoreFormat &&
-            this.includeFiles === otherExportParams.includeFiles &&
-            this.includeDeleted === otherExportParams.includeDeleted &&
-            this.includeHidden === otherExportParams.includeHidden
-        );
+        return sameExportParams(this, otherExportParams);
+    }
+}
+
+export function sameExportParams(params1?: ExportParams, params2?: ExportParams): boolean {
```

Comment:
Could we use a verb in this bool returning function please? I find them easier to read. Something like:
```suggestion
export function areEqual(params1?: ExportParams, params2?: ExportParams): boolean {
```

Reply:
Sure! thanks!